low-level/files/opt/seagate/sspl/setup/resource_map/storage.py

```diff
@@ -128,6 +136,39 @@ def get_controllers_info(self):
             data.append(controller_dict)
         return data
 
+    def get_platform_sensors(self):
+        sensor_list = ['temperature', 'current', 'voltage']
+        sensor_data = self.build_encl_platform_sensors_data(sensor_list)
+        return sensor_data
+
+    def build_encl_platform_sensors_data(self, platform_sensors):
+        sensors_data = {}
+        sensors_resp = self.get_realstor_encl_data('sensor-status')
+        if sensors_resp:
+            for platform_sensor in platform_sensors:
+                for sensor in sensors_resp['api-response']['sensors']:
+                    if sensor['sensor-type'].lower() == platform_sensor:
+                        single_senosr_data = {
+                          'uid': sensor.get('durable-id'),
+                          'fru': 'false',
+                          'last_updated':  int(time.time()),
+                          'health': {
+                            'status': sensor.get('status'),
+                            'description': '',
+                            'recommendation': '',
+                            'specifics': [
+                              {
+                                'value': sensor.get('value'),
+                                'sensor-name': sensor.get('sensor-name'),
```

Comment:
put sensor-name first.
Also feed items which we give through existing sensor, like threshold values will matter here

Reply:
Done.

Reply:
Threshold kv pairs i dont see still?

Reply:
For storage sensors it is not specified in schema. Threshold kv pairs only mentioned in server sensors.

Reply:
Ok, schema doesnt have everything btw. If we feel some other data can be useful, we can think of adding here. So if threshold data available, do add it

Reply:
It is not available.

Reply:
>           {
            "drawer-id-numeric": 255,
            "status": "OK",
            "container": "controllers",
            "enclosure-id": 0,
            "sensor-type": "Voltage",
            "durable-id": "sensor_volt_ctrl_B.1",
            "value": "2.03",
            "object-name": "sensor",
            "controller-id-numeric": 0,
            "container-numeric": 19,
            "controller-id": "B",
            "sensor-type-numeric": 2,
            "sensor-name": "Capacitor Cell 1 Voltage-Ctlr B",
            "drawer-id": "N/A",
            "status-numeric": 1
            },
            
Ok, thanks for sharing, it helps to clarify. then lets conclude here